website/source/guides/identity/authentication.html.md

```diff
@@ -10,27 +10,27 @@ description: |-
 # Authentication
 
 Before a client can interact with Vault, it must authenticate against an [**auth
-backend**](/docs/auth/index.html) to acquire a token. This token has policies attached so
+method**](/docs/auth/index.html) to acquire a token. This token has policies attached so
 that the behavior of the client can be governed.
 
 Since tokens are the core method for authentication within Vault, there is a
-**token** auth backend (often refer as **_token store_**). This is a special
-auth backend responsible for creating and storing tokens.
+**token** auth method (often refer as **_token store_**). This is a special
+auth method responsible for creating and storing tokens.
 
 ### Auth Backends
```

Comment:
Should this be titled "Auth Methods" since the rest of the terminology changed?

Reply:
I thought I fixed them all...but I must have missed those.  Thanks!